### ISSUE TITLE
Clarify and simplify setup options

### ISSUE THREAD

**Issue by nicksellen** [yunity/yunity-setup#11]
It seems it would be useful to have a clearer set of options, e.g. if you just want to work on the frontend, you have have everything (services + backend + sockets) running in docker containers.

If you are working on the backend, you probably will want to run the python server yourself (i.e. not via pm2) to get debugging/reloading.

I don't think we need to vagrant stuff anymore either. Docker is a better option now.

**Comment by nicksellen** [comment]
... and remove duplication between wiki and readme's (as per #12)

**Comment by stefaneidelloth** [comment]
... and please clarify if you use Trello (https://trello.com/b/xbljvnbQ/product) which seems to be up to date or github issues and waffle (https://waffle.io/yunity/yunity) as mentioned here: https://yunity.atlassian.net/wiki/display/YUN/Development+Process
 or both, trello and waffle, but for different purpose.

**Comment by nicksellen** [comment]
@stefaneidelloth we use both, if you want to get more involved then I suggest to join here https://project.yunity.org/join-the-team

Please don't use github issues as a place to ask questions about other topics, this issue is just focused on the setup process.

**Comment by stefaneidelloth** [comment]
So github is the the right place for issue tickets on the setup process?

**Comment by nicksellen** [comment]
Yes, if the issue is specific to that repo, and of a mostly technical nature.
